src/components/AssetEventsPanel/stories/basic.story.mdx

```diff
@@ -147,3 +151,31 @@ const handleAssetEventsLoaded = (events: CogniteEvent[]) => console.log(events);
     />
   </Story>
 </Preview>
+
+### Customer render of metadata.
+
+You can provide a custom function to render your metadata:
+
+```typescript
+import { EventAddonsProp, MetadataRenderer } from '@cognite/sdk';
```

Comment:
```suggestion
import { EventAddonsProp, MetadataRenderer } from '@cognite/gearbox';
```

this comes from gearbox, isn't it?

Reply:
ah, right, should be fixed

Reply:
btw I am not sure if it's exported even, can you change that? maybe we need a better name for it as well..

Reply:
Fixed!